packages/proxy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.13-alpine
+FROM node:lts-alpine
```

Comment:
I don't think you meant to change the container from nginx to node... I think you meant to change it to this?
FROM nginx:stable-alpine

Reply:
fixed, good catch!